client/state/notices/middleware.js

```diff
@@ -319,6 +320,10 @@ export const onBillingTransactionRequestFailure = ( { transactionId, error } ) =
 	} );
 };
 
+export const onJetpackConnectAuthorizeFailure = ( { message } ) => {
+	return errorNotice( message );
```

Comment:
The previous notice was a little less scary (yellow). Should this one be a `warningNotice` instead? Or perhaps we could have a `type` in addition to the `message` that would specify the notice type?